src/phase0/presets/mainnet.rs

```diff
@@ -24,11 +24,11 @@ pub const SLOTS_PER_HISTORICAL_ROOT: usize = 8192;
 pub const EPOCHS_PER_HISTORICAL_VECTOR: usize = 65536;
 pub const EPOCHS_PER_SLASHINGS_VECTOR: usize = 8192;
 pub const HISTORICAL_ROOTS_LIMIT: usize = 16_777_216;
-pub const VALIDATOR_REGISTRY_LIMIT: usize = 2usize.pow(40);
+pub const VALIDATOR_REGISTRY_LIMIT: usize = 2usize.saturating_pow(40);
 pub const BASE_REWARD_FACTOR: u64 = 64;
 pub const WHISTLEBLOWER_REWARD_QUOTIENT: u64 = 512;
 pub const PROPOSER_REWARD_QUOTIENT: u64 = 8;
-pub const INACTIVITY_PENALTY_QUOTIENT: u64 = 2u64.pow(26);
+pub const INACTIVITY_PENALTY_QUOTIENT: u64 = 2u64.saturating_pow(26);
```

Comment:
not sure why we changed these

Reply:
Yh! so this is due to the compilation target wasm32-unknown-unknown